Fix typo and style (#11260)
pingcap/docs: .github/ISSUE_TEMPLATE/change-request.md

2. Describe your suggestion or addition.

> 3. Provide some reference materials (documents, websites, and so on) if you could.

Reviewer: Neither "etc." or "and so on" is encouraged. [Ref](https://developers.google.cn/style/word-list#etc)

```suggestion
3. Provide some reference materials (such as documents and websites) if you could.
```
